controls.py

```diff
@@ -0,0 +1,78 @@
+import keyboard
+import pygame
+import sys
+
+class Controls:
+    def __init__(self, comms, exit_program):
+        self.comms = comms
+        self.exit_program = exit_program
+        
+        self.running = True
+        self.SCREEN_COLOR = (0, 0, 0)
+        self.SCREEN_SIZE = (100, 100)
+        
+        pygame.init()
+        self.screen = pygame.display.set_mode(self.SCREEN_SIZE)
+        self.screen.fill(self.SCREEN_COLOR)
+    
+    def on_trigger(self, key):
+        self.comms.read_send(key)
+        print("Sent")
+
+    def run(self):
+        while self.running:
+            for event in pygame.event.get():
+                if event.type == pygame.QUIT:
+                    self.running = False
+                
+                if not (pygame.key.get_pressed()[pygame.K_LSHIFT] or pygame.key.get_pressed()[pygame.K_RSHIFT]):
+                    if event.type == pygame.KEYDOWN:
+                        if event.key == pygame.K_w:
```

Comment:
These can be simplified to a for loop. Create a list of tuples, comparing the first element of the tuple then passing the second element of the tuple as an argument to `self.on_trigger`. 

So a tuple would look something along the lines of:

```
(pygame.K_d, "d")
```